sqlalchemy_hana/dialect.py

```diff
@@ -369,6 +369,10 @@ def visit_uuid(self, type_: types.TypeEngine[Any], **kw: Any) -> str:
         # SAP HANA has no UUID type, therefore delegate to NVARCHAR(32)
         return self._render_string_type(type_, "NVARCHAR", length_override=32)
 
+    # pylint: disable=invalid-name
+    def visit_JSON(self, type_: types.TypeEngine[Any], **kw: Any) -> str:
+        return self.visit_NCLOB(type_, **kw)
```

Comment:
I think this is only half of the needed code. Please add a test really doing something that this is all we need

Reply:
Could you elaborate a bit what kind of tests / references you wnat to add, given I added one test in the test_types?

Reply:
I would like to see an e2e-test. Meaning you create a table with a JSON column, you insert some data, maybe modify it and select it

Reply:
Isn't that exactly whats happening in _TypeBaseTest in test_types.py? 
<img width="786" alt="image" src="https://github.com/SAP/sqlalchemy-hana/assets/218582/74865a75-6eec-4a7b-87df-4c10b5aa1e13">

Arguably one could add another test case that modifies the data in between insert and read.

Reply:
Sorry, I missed that you extended this class. Thanks!